bigtable/google/cloud/bigtable/table.py

```diff
@@ -340,7 +350,7 @@ def read_rows(self, start_key=None, end_key=None, limit=None,
         data_client = self._instance._client.table_data_client
         return PartialRowsData(
             data_client.transport.read_rows,
-            request_pb)
+            request_pb, retry)
```

Comment:
We need an assertion in `TestTable.test_read_rows` for the default case, and another test case exercising the override.

Reply:
Done. Added a new test "test_yield_retry_rows_data".

Reply:
@sumit-ql Did you push that change?  I don't see any new code.

Reply:
@tseaver My bad , forgot to push the changes yesterday.Pushed now.
Please review.

Reply:
@tseaver I have made the changes in unit tests as per your comments, please review.